intern-book-app/src/app/pages/list/list.component.ts

```diff
@@ -9,6 +10,25 @@ import { Book } from 'src/app/types/book';
 export class ListComponent implements OnInit {
   title = '読んだ本';
 
+  book: Book = {
+    name: '',
+    detail: '',
+    evaluation: 0,
+  };
+
+  formReset() {
+    //各パラメータに初期値を代入
+    (this.book.name = ''), (this.book.detail = ''), (this.book.evaluation = 0);
+  }
```

Comment:
`formReset`より`resetForm`の方がいいかも！
あと不思議な実装してるね…
```suggestion
  formReset() {
    //各パラメータに初期値を代入
    this.book = {
      name: '',
      detail: '',
      evaluation: 0,
    };
  }
```